### PR TITLE
docker GPU Api Update

### PR DESCRIPTION
**Pull request by gouzil**
To be verified

### CODE REVIEW

**Comment by gouzil** [comment]
| cmd      | use --gpus | image | Meets expectations |  remarks |
| ----------- | ----------- | ----------- | ----------- | ----------- |
| docker run --name paddle_docker_temp1 -it -v $PWD:/paddle registry.baidubce.com/paddlepaddle/paddle:2.3.2-gpu-cuda10.2-cudnn7 /bin/bash  | false | registry.baidubce.com/paddlepaddle/paddle:2.3.2-gpu-cuda10.2-cudnn7  |   | test1|
| docker run --name paddle_docker_temp1 -it -v $PWD:/paddle registry.baidubce.com/paddlepaddle/paddle:2.3.2 /bin/bash   | false |  registry.baidubce.com/paddlepaddle/paddle:2.3.2    |  | test2 |
| docker run --name paddle_docker_temp1 --gpus all  -it -v $PWD:/paddle registry.baidubce.com/paddlepaddle/paddle:2.3.2 /bin/bash | true |  registry.baidubce.com/paddlepaddle/paddle:2.3.2 |  | test3 |
|docker run --name paddle_docker -it -v $PWD:/paddle registry.baidubce.com/paddlepaddle/paddle:2.3.2-gpu-cuda11.2-cudnn8  /bin/bash | true | registry.baidubce.com/paddlepaddle/paddle:2.3.2-gpu-cuda11.2-cudnn8 |  | test4 |

## test1
test1

## test2
test2

## test3
test3

## test4
test4

**Comment by gouzil** [comment]
| cmd      | use --gpus | Meets expectations |  remarks |
| ----------- | ----------- | ----------- | ----------- |
| docker run --name paddle_docker_temp1 -it --gpus all -v $PWD:/paddle registry.baidubce.com/paddlepaddle/paddle:2.3.2-gpu-cuda11.2-cudnn8 /bin/bash  | true |   | test1 |
| docker run --name paddle_docker_temp1 -it  -v $PWD:/paddle registry.baidubce.com/paddlepaddle/paddle:2.3.2-gpu-cuda11.2-cudnn8 /bin/bash  | false |    | test2 |

# test1

截屏2022-10-27 下午7 57 06

# test2

截屏2022-10-27 下午8 01 30

**Comment by dingjiaweiww** [comment]
根据上述测试结果，docker run如果不添加--gpus 参数，则无法正常调用GPU，参考：https://docs.docker.com/config/containers/resource_constraints/#gpu

**Review by XieYunshen — APPROVED** [review]
LGTM

File: docs/install/docker/linux-docker.md [hunk 1]
```diff
@@ -8,7 +8,41 @@
 
 - 在本地主机上[安装 Docker](https://hub.docker.com/search/?type=edition&offering=community)
 
-- 如需在 Linux 开启 GPU 支持，请[安装 nvidia-docker](https://github.com/NVIDIA/nvidia-docker)
+- 如需在 Linux 开启 GPU 支持, 需提前[安装 nvidia-container-toolkit](https://github.com/NVIDIA/nvidia-docker)
```
Comment (pangyoki) [inline docs/install/docker/linux-docker.md:11]: 这个链接需要更新吗？

Comment (gouzil) [reply docs/install/docker/linux-docker.md:11]: 不需要，他们是同个东西

**Review by pangyoki — APPROVED** [review]
LGTM

**Pull request merged**